---
layout: default
title: Jose Barba-Montoya
---

	<div class="research">		
					<h1>Research</h1>
					<p style="text-align:center;"><img src="/img/photo15.jpg" height="200" width="678"></p>
					<h3>Google Scholar profile</h3>
					<p><a href="https://scholar.google.co.uk/citations?hl=en&user=zLHdW1IAAAAJ"> scholar.google.co.uk/citations?hl=en&user=zLHdW1IAAAAJ</a>		
					
					<h3>Publications</h3>
<p><b>Effect of local calibrations overwhelm estimated divergence times.</b><br>
Kohli M, <b>Barba-Montoya J</b>, Ware J, Evangelista DA. 2024. (Submitted).</a>

<p><b>A legacy of genetic entanglement with wolves shapes modern dogs.</b><br>
Lin AT, Scarsbrook L, <b>Barba-Montoya J</b>, Liu HL, Larson G, Kistler L. 2024. (In review).</a>
		
<p><b>Integrating phylogenies with chronology to assemble the Tree of Life.</b><br>
<b>Barba-Montoya J</b>, Craig JM, Kumar S. 2024. (In revision, preprint available. <a href="https://www.biorxiv.org/content/10.1101/2024.07.17.603989v1">DOI: doi.org/10.1101/2024.07.17.603989).</a>

<p><b>Completing a molecular timetree of apes and monkeys.</b><br>
Craig JM, Bamba GL, <b>Barba-Montoya J</b>, Hedges SB, Kumar S. 2023. Frontiers in Bioinformatics, 3:1284744. <a href="https://www.frontiersin.org/articles/10.3389/fbinf.2023.1284744">DOI: doi.org/10.3389/fbinf.2023.1284744.</a>

<p><b>Molecular timetrees using relaxed clocks and uncertain phylogenies.</b><br>
<b>Barba-Montoya J</b>, Sharma S, Kumar S. 2023. Frontiers in Bioinformatics, 3:1225807. <a href="https://www.frontiersin.org/articles/10.3389/fbinf.2023.1225807/full">DOI: doi.org/10.3389/fbinf.2023.1225807.</a>

<p><b>Assessing rapid relaxed-clock methods for phylogenomic dating.</b><br>
<b>Barba-Montoya J</b>, Tao Q, Kumar S. 2021. Genome Biology and Evolution, 13(11). <a href="https://doi.org/10.1093/gbe/evab251">DOI: doi.org/10.1093/gbe/evab251.</a>

<p><b>Data-driven speciation tree prior for better species divergence times in calibration-poor molecular phylogenies.</b><br>
Tao Q, <b>Barba-Montoya J</b>, Kumar S. 2021. Bioinformatics, 37(S1): i102–i110 <a href=" https://doi.org/10.1093/bioinformatics/btab307">DOI: doi.org/10.1093/bioinformatics/btab307.</a>   

<p><b>Molecular and morphological clocks for estimating evolutionary divergence times.</b><br>
<b>Barba-Montoya J</b>, Tao Q, Kumar S. 2021. BMC Ecology and Evolution, 21(83). <a href="https://doi.org/10.1186/s12862-021-01798-6">DOI: doi.org/10.1186/s12862-021-01798-6.</a>   

<p><b>Molecular dating for phylogenies containing a mix of populations and species by using Bayesian and RelTime approaches.</b><br>
Mello B. Tao Q, <b>Barba-Montoya J</b>, Kumar S. 2021. Molecular Ecology Resources, 21: 122–136. <a href="https://doi.org/10.1111/1755-0998.13249">DOI: doi.org/10.1111/1755-0998.13249.</a>

<p><b>Using a GTR+Γ substitution model for dating sequence divergence when stationarity and time-reversibility assumptions are violated.</b><br>
<b>Barba-Montoya J</b>, Tao Q, Kumar S. 2020. Bioinformatics, 36(S2): i884–i894. <a href="https://doi.org/10.1093/bioinformatics/btaa820">DOI: doi.org/10.1093/bioinformatics/btaa820.</a>

<p><b>Relative efficiencies of simple and complex substitution models in estimating divergence times in phylogenomics.</b><br>
Tao Q, <b>Barba-Montoya J</b>, Huuki LA, Durnan MK, Kumar S. 2020. Molecular Biology and Evolution, 37(6): 1819–1831. <a href="https://doi.org/10.1093/molbev/msaa049">DOI: doi.org/10.1093/molbev/msaa049.</a>

<p><b>Constraining uncertainty in the timescale of angiosperm evolution and the veracity of a Cretaceous Terrestrial Revolution.</b><br>
<b>Barba-Montoya J</b>, dos Reis M, Schneider H, Donoghue PCJ, Yang Z. 2018. New Phytologist, 218: 819–834. <a href="https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.15011">DOI: doi.org/10.1111/nph.15011.</a>

<p><b>Using phylogenomic data to explore the effects of relaxed clocks and calibration strategies on divergence time estimation: Primates as a test case.</b><br>
dos Reis M, Gunnell GF, <b>Barba-Montoya J</b>, Wilkins A, Yang Z, Yoder AD. 2018. Systematic Biology, 67(4): 594–615. <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy001/4802240">DOI: doi.org/10.1093/sysbio/syy001.</a>

<p><b>Comparison of different strategies for using fossil calibrations to generate the time prior in Bayesian molecular clock dating.</b><br>
<b>Barba-Montoya J</b>, dos Reis M, Yang Z. 2017. Molecular Phylogenetics and Evolution, 114: 386–400. <a href="https://www.sciencedirect.com/science/article/pii/S1055790317303044?via%3Dihub"> DOI: doi.org/10.1016/j.ympev.2017.07.005.</a>

<p><b>¿Por qué tantos cactos columnares? El papel de los murciélagos nectarívoros en su diversificación.</b><br>
<b>Barba-Montoya J</b>, Magallón S. 2012.  Oikos=, 6: 17-19. <a href="https://www.biodiversitylibrary.org/item/201871#page/17"> biodiversitylibrary.org/item/201871#page/17.</a>


					<h3>Dissertations</h3>
<p>Ph.D. Thesis (University College London, UK), 2017: <b>Bayesian molecular clock dating and the divergence times of angiosperms and primates.</b> Supervised by Professor Ziheng Yang. <a href="http://discovery.ucl.ac.uk/1565539/"> discovery.ucl.ac.uk/1565539/.</a>

<p>M.Sc. Thesis (National Autonomous University of Mexico), 2012: <b>Diversification of Pachycereeae (Cactaceae) due to pollination syndromes.</b> Supervised by Professor Susana Magallón. <a href="https://ru.dgb.unam.mx/handle/20.500.14330/TES01000678067"> ru.dgb.unam.mx/handle/20.500.14330/TES01000678067/.</a> 

<p>B.Sc. Thesis (National Autonomous University of Mexico), 2009: <b>Origin and evolution of tribe Pachycereeae (Cactaceae).</b> Supervised by Professor Susana Magallón. <a href="https://ru.dgb.unam.mx/handle/20.500.14330/TES01000645528"> ru.dgb.unam.mx/handle/20.500.14330/TES01000645528/.</a>

<p style="text-align:center;"><img src="/img/photo6.jpg" height="250" width="250"></p>	

</div>
